crates/stdarch-verify/tests/x86-intel.rs

```diff
@@ -471,6 +471,12 @@ fn matches(rust: &Function, intel: &Intrinsic) -> Result<(), String> {
             continue;
         }
 
+        // Some AMD CPUs support VAES without AVX512, even though the Intel
+        // documentation states that those instructions require AVX512VL.
+        if *cpuid == "AVX512VL" && intel.cpuid.contains(&"VAES".to_string()) {
```

Comment:
`.contains(&"VAES".to_string())` is a bit goofy looking but it's test only and probably clearer than the alternative so it's fine.